packages/auth/migrations/20220504163005_create_access_token_table.js

```diff
@@ -2,7 +2,7 @@ exports.up = function (knex) {
   return knex.schema.createTable('accessTokens', function (table) {
     table.uuid('id').notNullable().primary()
     table.string('value').notNullable().unique()
-    table.string('managementId').notNullable()
+    table.string('managementId').notNullable().unique()
```

Comment:
Why do we introduce that? The `id` is already unique, so why adding another unique id?

Reply:
I figured it would be the right practice to expose an external id instead of an internal database id. I should update it to a `.uuid()` instead of a `.string()` though since in practice it's always a uuid/v4.

Reply:
>I should update it to a .uuid() instead of a .string() though since in practice it's always a uuid/v4.

bump